src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Ball{
+    public Color color;
+    public int number;
```

Comment:
```suggestion
    private Color color;
    private int number;
```
create a constructor